### PR TITLE
App delivery

### PR DESCRIPTION
**Pull request by haavamoa**
### Description of Change

- Added build script for delivering Components app to AppCenter
- Added in app API for checking version of AppCenter
- Breaking change: Changed the ItemPicker base class to remove unessesary properties.
- Breaking change: Changed ItemPickers Title property to Placeholder
- Added ItemSpacing for CollectionView
- Added a default ItemSpacing for CollectionView
- Added VerticalStackLayout
- Added default Spacing for VerticalStackLayout
- Added StringFormat markup extension to easily use a static string format with an hard argument in XAML.
- Cleaned up components app
- Made sure Components app samples are alphabetical

### Todos
- [X] I have tested on an Android device.
- [X] I have tested on an iOS device.
- [ ] I have supported accessibility

### CODE REVIEW

File: src/app/Components/Components.csproj [hunk 1]
```diff
@@ -23,15 +23,30 @@
         <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">30.0</SupportedOSPlatformVersion>
     </PropertyGroup>
 
+    <PropertyGroup>
+        <DIPSIncludeIcons>False</DIPSIncludeIcons>
```
Comment (Vetle444) [inline src/app/Components/Components.csproj:27]: Hmm, ska denna settes t false?

File: src/app/Components/Components.csproj [hunk 2]
```diff
@@ -23,15 +23,30 @@
...
+    </PropertyGroup>
     <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net7.0-ios|AnyCPU'">
       <CreatePackage>false</CreatePackage>
+      <CodesignProvision>Automatic</CodesignProvision>
+      <CodesignKey>iPhone Developer</CodesignKey>
+    </PropertyGroup>
```
Comment (Vetle444) [inline src/app/Components/Components.csproj:33]: Ska detta værmæ?

File: src/app/Components/Components.csproj (hunk 1 above)
Comment (haavamoa) [reply src/app/Components/Components.csproj:27]: Nei, fjerner.

File: src/app/Components/Components.csproj (hunk 2 above)
Comment (haavamoa) [reply src/app/Components/Components.csproj:33]: Ja, ellers forsøker appen å signere med in house signing.

**Review by Vetle444 — APPROVED** [review]

**Pull request merged**